pkg/gofr/http/responder.go

```diff
@@ -65,6 +70,22 @@ func (r Responder) HTTPStatusFromError(err error) (status int, errObj interface{
 	}
 }
 
+// HTTPStatusFromRequestMethod maps request method to HTTP status codes.
+func (r Responder) HTTPStatusFromRequestMethod() int {
```

Comment:
This method is unnecessary addition. Could be included in `HTTPStatusFromErr` method itself.